karma.config.js

```diff
@@ -36,7 +36,7 @@ const rollupConfig = () => ({
   },
   plugins: [
     replace({
-      'process.env.NODE_ENV': JSON.stringify('production'),
+      'process.env.NODE_ENV': JSON.stringify('test'),
```

Comment:
Tell me please, why changed node var?

Reply:
Because `@testing-library/react` use `act(...)`, that does not work in prod env.

https://github.com/uploadcare/react-widget/runs/501054004?check_suite_focus=true#step:5:34